### DISCUSSION TITLE
GitHub Education for Students benefits

### DISCUSSION THREAD

**Discussion by Darasimi333** [community/community#136484] [GitHub Education]
### Hi everyone,

Hello GitHub Education common 
I want to complain of my GitHub Education student benefits that not be appear.

Please, help me look in to it

**Comment by davevad93** [comment]
Hi @Darasimi333  ,

How long have you been waiting? The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Darasimi333 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
